lib/client/js/ui/BeansView.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * BeansView class that extends from the base Control class to
+ * dynamically build and update a UI container that will
+ * house the main menu and content sections.
+ *
+ * @class
+ */
+var BeansView = (function () {
+    /**
+     * @alias BeansView
+     */
+    function BeansView() {
+        /** @member {Object} */
+        /** @member {Object} */
+        this._Menu = new Menu();
+        this._Content = new Content();
+    }
+
+    BeansView.prototype = new Control();
+
+    /**
+     * A "pure" view method, which is to say that mutation of the
+     * BeansView instance should not happen here. It just takes what's in
+     * the instance and uses it to provide a view. This should only be
+     * used internally. This will be used indirectly by consumers through
+     * the render method.
```

Comment:
I think we're gonna do away with the render method so we can remove the last couple clauses.